src/core/src/opsets/opset.cpp

```diff
@@ -48,6 +48,26 @@ ov::Node* ov::OpSet::create_insensitive(const std::string& name) const {
     return m_factory_registry.create(type_info_it->second);
 }
 
+#define _OPENVINO_REG_OPSET(OPSET) opset_map[#OPSET] = ov::get_##OPSET;
+
+const std::map<std::string, std::function<const ov::OpSet&()>>& ov::get_avalable_opsets() {
```

Comment:
will it affect CC? 
here we introduce one more points which holds other code

Reply:
No it shouldn't affect. Here we hold only constrctors of opsets. But CC works with insert operations